adage/adage/config.py.template

```diff
@@ -7,6 +7,10 @@
 # and replace with values tailored to your deployment.
 import random
 
+import os
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
```

Comment:
I just realized another approach would be to put CI-specific configuration into a separate section of the CONFIG. I did that for Codeship (see CI_CONFIG). So we can revert the changes to TEST_CONFIG and set up whatever we want in CIRCLECI_CONFIG. That would keep the various deployment scheme settings isolated.